test/helper/journey/cancel_from_restart_journey.go

```diff
@@ -28,6 +29,9 @@ func CancelFromRestartJourney(t *testing.T, cluster *docker.DockerCompose, nodeN
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
 	defer cancel()
 
+	os.Setenv("ENABLE_CLEANUP_UNFINISHED_BACKUPS", "true")
+	defer os.Unsetenv("ENABLE_CLEANUP_UNFINISHED_BACKUPS")
```

Comment:
you can use just `t.Setev("ENABLE_CLEANUP_UNFINISHED_BACKUPS", "true")` instead of those 2 lines, `t` automatically cleans up after itself